nrf_security/src/psa/Kconfig

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig PSA_NATIVE_ITS
+	bool "PSA native Internal Trusted Storage"
+	depends on CC310_BACKEND
+
+if PSA_NATIVE_ITS
+
+choice
+	prompt "Backend selection for PSA native ITS"
+	depends on PSA_NATIVE_ITS
```

Comment:
Nit: You don't need this line when this is inside the `if PSA_NATIVE_ITS`. They are equivalent.